portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,15 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function darkMode() {
+  var element = document.body;
+  element.classList.toggle("dark-mode");
+  const modeToggle = document.getElementById('modeToggle');
+  if(modeToggle.innerText == "Dark"){
+    modeToggle.innerText = "Light";
+  }
+  else{
+    modeToggle.innerText = "Dark";
+  }
+}
```

Comment:
nit: Add a newline. Always a good idea to have an empty newline at end of file. 
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline